dask/bytes/local.py

```diff
@@ -33,10 +33,14 @@ def __init__(self, **storage_options):
 
     def glob(self, path):
         """For a template path, return matching files"""
+        if path.startswith('file://'):
+            path = path[len('file://'):]
```

Comment:
It would be nice to defend others who make more of these things in the future if we can.